libgalois/include/katana/PropertyGraph.h

```diff
@@ -403,8 +410,14 @@ class KATANA_EXPORT PropertyGraph {
 
   const GraphTopology& topology() const { return topology_; }
 
+  /// Add Node properties that do not exist in the current graph
   Result<void> AddNodeProperties(const std::shared_ptr<arrow::Table>& props);
+  /// Add Edge properties that do not exist in the current graph
   Result<void> AddEdgeProperties(const std::shared_ptr<arrow::Table>& props);
+  /// If column name exists, replace it, otherwise insert it
```

Comment:
nit: the above comments do not refer to properties as columns. I'd prefer "property name" instead.